flytekit/core/base_task.py

```diff
@@ -236,7 +250,13 @@ def _local_execute(self, ctx: FlyteContext, **kwargs) -> Union[Tuple[Promise], P
         )
         input_literal_map = _literal_models.LiteralMap(literals=kwargs)
 
-        outputs_literal_map = self.dispatch_execute(ctx, input_literal_map)
+        # TODO: improve comment
+        # if metadata.cache is set, check memoized version including cache_version
```

Comment:
cc @wild-endeavor 
FYI, Yee is working on some updates to this part of the code as well

Reply:
nope